### PR TITLE
Add ShineWiFi-X/S Datalogger Reconfiguration Guide

### PR DESCRIPTION
**Pull request by doe81**
Added a detailed step-by-step guide for reconfiguring the ShineWiFi-X/S datalogger using the ShinePhone app.

### CODE REVIEW

File: docs/setup/datalogger.md [hunk 1]
```diff
@@ -24,6 +24,51 @@ After:
 
 After this your ShineLink/Wifi Dongle will start pushing data to Grott (and optionally more frequently than default also).
 
+## ShineWiFi-X/S Re-configuration
+
+To reconfigure your ShineWiFi-X/S datalogger, you need to use the ShinePhone app and have your datalogger already added to your plant. (This guide and photos are based on Android.)
+
+1. **Start the ShinePhone App:**
```
Comment (muppet3000) [inline docs/setup/datalogger.md:31]: Please can you change all of the bullet point numbers to `1.` Github will auto-re-number them when the page is published, it means that if there's any modifications in the future nobody has to go through and change all the numbers.

**Comment by muppet3000** [comment]
Thanks for this PR, it's exactly what I was hoping someone would submit one day!

I've just added 1 comment, if you can tweak that I'm happy to merge!

File: docs/setup/datalogger.md (hunk 1 above)
Comment (doe81) [reply docs/setup/datalogger.md:31]: They are now updated as requested. 👍

**Comment by doe81** [comment]
I'm happy to contribute! I took the time to take the screenshots since my datalogger hade reset to default setting somehow.

**Pull request merged**